tests/testthat/test-remove_brackets.R

```diff
@@ -0,0 +1,32 @@
+test_that("remove_brackets() works in simple cases", {
+
+  expect_identical(
```

Comment:
I was hoping to see these tests also relate to the integration of `remove_brackets()` with the other functions. Maybe we can add this at a later stage.